Sprint-3/revise/implement/get-ordinal-number.test.js

```diff
@@ -2,3 +2,62 @@
 
 // continue testing and implementing getOrdinalNumber for additional cases
 // Write your tests using Jest - remember to run your tests often for continual feedback
+const getOrdinalNumber = require("./get-ordinal-number");
+
+describe("getOrdinalNumber", () => {
+  test("returns 1st for 1", () => {
+    expect(getOrdinalNumber(1)).toBe("1st");
+  });
+
+  test("returns 2nd for 2", () => {
+    expect(getOrdinalNumber(2)).toBe("2nd");
+  });
+
+  test("returns 3rd for 3", () => {
```

Comment:
Can you think of a description here that adds MORE information instead of just restating the assertion?

What is the underlying pattern you are looking for? What do these tests really mean?

Reply:
I've updated the test descriptions to be more descriptive and explain the underlying patterns I'm testing for. Each test now describes the expected ordinal number and includes details about edge cases like 11th, 12th, and 13th.